explore/src/main/scala/explore/targeteditor/ElevationPlotSection.scala

```diff
@@ -147,22 +150,31 @@ object ElevationPlotSection:
           <.div(ExploreStyles.ElevationPlot)(
             opt.range match
               case PlotRange.Night    =>
+                val coords: CoordinatesAtVizTime =
+                  props.tracking
+                    .at(dateView.get.atStartOfDay.toInstant(ZoneOffset.UTC))
+                    .getOrElse(CoordinatesAtVizTime(props.tracking.baseCoordinates))
+
                 ElevationPlotNight(
-                  props.coords,
+                  coords,
                   props.visualizationTime,
                   windowsNetExcludeIntervals,
                   props.pendingTime,
                   options
                 )
               case PlotRange.Semester =>
+                val coords: CoordinatesAtVizTime =
+                  props.tracking
+                    .at(semesterView.get.start.atSite(siteView.get).toInstant)
```

Comment:
We maybe want to use middle of semester here

Reply:
I doubt it makes much of a difference for sidereal targets. For non-sidereal targets, I expect we will have to implement a different way of computing this, since coordinates can change each day (or even faster).